services/metadata/backend/api/middleware/BUILD.bazel

```diff
@@ -9,7 +9,7 @@ go_library(
     importpath = "github.com/dasch-swiss/dasch-service-platform/services/metadata/backend/api/middleware",
     visibility = ["//visibility:public"],
     deps = [
-        "//services/metadata/backend/pkg/metric",
+        "//shared/go/pkg/metric",
```

Comment:
why shared?

Reply:
just a name we use in the c++ repo to clearly denote that stuff underneath is shared between services/components.

Reply:
Sorry maybe the question was set wrong. The name is obvious, but should we have it outside services or rather inside?

Reply:
Just checked that BS-29 is actually about that :D, so there must be reason behind ;)

Reply:
ah, ok, that is another question. The way it is named now `shared/go` was done with the idea in mind to have something like `shared/js` or even `shared/svelte`.

@mpro7 @BalduinLandolt 
I mentioned in the chat, that I have the feeling that the folder structure is not ideal. I have added, in another branch, `services/resource/backend` which will be the microservice for the resource and ontology API. There is not going to be one frontend for this, but rather a few WebComponents that will access this service. Maybe we need a `components` folder in root, under which all the frontend components will live? Then we would have in the root folder services, shared, and components.

Reply:
Like @BalduinLandolt mentioned, go for what you have now, then we can actually redesign and change it at any time.